src/pages/database/DatabaseContainer.tsx

```diff
@@ -7,25 +7,46 @@ import Database, { DatabaseProps } from './Database';
 
 // TODO: Clean up the various type exports...
 // `blocks` is excluded here as we do not need it when displaying Database for now.
+export type Id = string;
+export type Title = string;
+export type CurrentView = string;
+
 export type Note = {
   userId: string;
   databaseId: string;
   id: string;
   title: string;
+  blocks: [
+    {
+      id: string;
+      html: string;
+      tag: string;
+    }
+  ];
   creationDate: string;
   latestUpdate: string;
 };
```

Comment:
can shift this into notepage.tsx

i previously left this here as notepage was incomplete

Reply:
For rendering preview of blocks in board view